app/views.py

```diff
@@ -291,13 +295,33 @@ def taskView(request, projectID, taskID):
         submitLink = f"/project/{projectID}/kanban/task/{taskID}/"
         exitLink = f"/project/{projectID}/kanban"
         deleteLink = f"/project/{projectID}/kanban"
-        
+    
+    # Needed for both post + get
+    data, statusCode = ProjectsAPIView.getProjects(
+        {"id": projectID}, getAuthorizationToken(request)
+    )
+    if statusCode != status.HTTP_200_OK:
+        print(f"Project fetch failed: {data.get('message')}")
+        return HttpResponseServerError(f"An error occurred: {data.get('message')}")
+    projectData = data[0]
+
+    userData, statusCode = UsersAPIView.getUsers(
+        [{"id": id} for id in projectData.get("userIDs")], getAuthorizationToken(request))
+    if statusCode != status.HTTP_200_OK:
+        print(f"Users fetch failed: {data.get('message')}")
+        return HttpResponseServerError(f"An error occurred: {data.get('message')}")
+
+    
+    assigneeChoices = [(data.get('id'), data.get('username')) for data in userData]
+
```

Comment:
`data` is already defined so we should probably change this reference variable name

```suggestion
    
    assigneeChoices = [(userData.get('id'), userData.get('username')) for userData in userDataList]

```